api/tests/routes/backoffice/offers_test.py

```diff
@@ -113,7 +114,7 @@ class ListOffersTest(GetEndpointHelper):
     # - fetch session (1 query)
     # - fetch user (1 query)
     # - fetch offers with joinedload including extra data (1 query)
-    expected_num_queries = 3
+    expected_num_queries = 4
```

Comment:
Il faut rajouter un commentaire qui dit d'où vient la query en plus :)

Reply:
Je devine que c'est probablement le Feature Flag, à confirmer par @ghaliela .
```suggestion
    # - check Feature Flag: ENABLE_PRO_TITELIVE_MUSIC_GENRES
    expected_num_queries = 4
```

Reply:
Oui bien sûr

Reply:
:+1: